exporter/collector/integrationtest/cmd/recordfixtures/main.go

```diff
@@ -54,12 +54,23 @@ func main() {
 	startTime := endTime.Add(-time.Second)
 	t := &FakeTesting{}
 
-	recordLogs(ctx, t, endTime)
-	recordMetrics(ctx, t, startTime, endTime)
-	recordTraces(ctx, t, startTime, endTime)
+	// track the output fixtures to catch tests overwriting another test's fixtures
+	outputFixtures := make(map[string]string)
```

Comment:
nit might be cleaner to do this check in a test e.g. add a new `exporter/collector/integrationtest/testcases/test_testcases.go`

Reply:
Keeping this here throws an error and stops fixture generation at the source of the problem. Eg, I run `make fixtures` and immediately see that one of my fixtures is badly copy-pasted because the command fails.

If we move this to a separate test, that decouples this warning from what it's trying to warn about. imo this is more of a runtime error in the `recordfixtures` cmd that should stop execution.

Reply:
Yeah, I don't really feel like renaming all of our fixtures in this PR especially since the scope of the changes here are already beyond the original goal. I'll open another issue to track that

Reply:
https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/615